src/OpenSearch.Client/_Generated/Requests.Tasks.cs

```diff
@@ -0,0 +1,233 @@
+/* SPDX-License-Identifier: Apache-2.0
+*
+* The OpenSearch Contributors require contributions made to
+* this file be licensed under the Apache-2.0 license or a
+* compatible open source license.
+*/
+/*
+* Modifications Copyright OpenSearch Contributors. See
+* GitHub history for details.
+*
+*  Licensed to Elasticsearch B.V. under one or more contributor
```

Comment:
Do we need elastic license in a new file?

Reply:
I believe it's still required due to the generator code/templates themself still being licensed to ES. Never got a definitive answer saying safe to drop.

cc: @dblock 

https://github.com/opensearch-project/opensearch-net/issues/73

Reply:
I think it’s safe to keep.